model/const.go

```diff
@@ -218,6 +218,10 @@ const (
 	ListObjectsDelimiterQuery = "delimiter"
 	// ListObjectsPrefixQuery defines limits the response to keys that begin with the specified prefix
 	ListObjectsPrefixQuery = "prefix"
+	// GetBucketMetaByNameQuery defines get bucket metadata query, which is used to route request
+	GetBucketMetaByNameQuery = "bucket-meta"
+	// GetObjectMetaByNameQuery defines get object metadata query, which is used to route request
+	GetObjectMetaByNameQuery = "object-meta"
```

Comment:
I think it is also suitable to remove ByName, because in our scene, the default is indexed by Name. (Maybe only ByObjectID needs to be pointed out).

Reply:
Make sense, I renamed them and related methods, thanks for the suggestion!